Workbooks/SCOM managed instance/SCOM managed instance/SCOM managed instance.workbook

```diff
@@ -0,0 +1,375 @@
+                    "color": "green"
+                  },
+                  {
+                    "seriesName": "2",
+                    "label": "Warning",
+                    "color": "yellow"
+                  },
+                  {
+                    "seriesName": "100",
+                    "label": "Unplanned maintenance"
+                  },
+                  {
+                    "seriesName": "101",
+                    "label": "Planned maintenance"
+                  }
+                ]
+              }
+            },
+            "customWidth": "35",
+            "name": "VM health state pie chart",
+            "styleSettings": {
+              "maxWidth": "35"
+            }
+          },
+          {
+            "type": 3,
+            "content": {
+              "version": "KqlItem/1.0",
+              "query": "WorkloadHealthState_CL\r\n| where MonitorName == 'System.Health.EntityState'\r\n| extend HealthStatus = iif(HealthState == 0, \"Unmonitored\", iif(HealthState == 1, \"Healthy\", iif(HealthState == 2, \"Warning\", iif(HealthState == 3, \"Error\", iif(HealthState == 100, \"Unplanned maintenance\", iif(HealthState == 101, \"Planned maintenance\", \"Other\"))))))\r\n| summarize arg_max(TimeGenerated, HealthStatus) by ComputerName",
+              "size": 0,
+              "title": "Health state of VMs",
+              "timeContext": {
+                "durationMs": 86400000
+              },
+              "queryType": 0,
+              "resourceType": "microsoft.operationalinsights/workspaces",
+              "crossComponentResources": [
+                "{LogAnalyticsWorkspace}"
+              ],
+              "visualization": "table",
+              "gridSettings": {
+                "sortBy": [
+                  {
+                    "itemKey": "TimeGenerated",
+                    "sortOrder": 2
+                  }
+                ],
+                "labelSettings": [
+                  {
+                    "columnId": "ComputerName",
+                    "label": "VM Name"
+                  },
+                  {
+                    "columnId": "HealthStatus",
+                    "label": "Health State"
+                  }
+                ]
+              },
+              "sortBy": [
+                {
+                  "itemKey": "TimeGenerated",
+                  "sortOrder": 2
+                }
+              ],
+              "tileSettings": {
+                "showBorder": false
+              },
+              "mapSettings": {
+                "locInfo": "LatLong"
+              }
+            },
+            "customWidth": "65",
+            "name": "VM Names with health state",
+            "styleSettings": {
+              "maxWidth": "65",
+              "showBorder": true
+            }
+          }
+        ]
+      },
+      "name": "group - 5",
+      "styleSettings": {
+        "showBorder": true
+      }
+    }
+  ],
+  "fallbackResourceIds": [
```

Comment:
strange that the build verification doesn't catch this: templates shouldn't have fallbackResourceIds, you should remove this

Reply:
Done